php/blocks/controls/class-post.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Post control.
+ *
+ * @package   Block_Lab
+ * @copyright Copyright(c) 2018, Block Lab
+ * @license http://opensource.org/licenses/GPL-2.0 GNU General Public License, version 2 (GPL-2.0)
+ */
+
+namespace Block_Lab\Blocks\Controls;
+
+/**
+ * Class Post
+ */
+class Post extends Control_Abstract {
+
+	/**
+	 * Control name.
+	 *
+	 * @var string
+	 */
+	public $name = 'post';
+
+	/**
+	 * Field variable type.
+	 *
+	 * @var string
+	 */
+	public $type = 'integer';
+
+	/**
+	 * Post constructor.
+	 */
+	public function __construct() {
+		parent::__construct();
+		$this->label = __( 'Post', 'block-lab' );
+	}
+
+	/**
+	 * Register settings.
+	 *
+	 * @return void
+	 */
+	public function register_settings() {
+		$this->settings[] = new Control_Setting(
+			array(
+				'name'     => 'help',
+				'label'    => __( 'Help Text', 'block-lab' ),
+				'type'     => 'text',
+				'default'  => '',
+				'sanitize' => 'sanitize_text_field',
+			)
+		);
+		$this->settings[] = new Control_Setting(
+			array(
+				'name'     => 'placeholder',
+				'label'    => __( 'Placeholder Text', 'block-lab' ),
+				'type'     => 'text',
+				'default'  => '',
+				'sanitize' => 'sanitize_text_field',
+			)
+		);
```

Comment:
It would be really cool to include a dropdown list of different post types here. It wouldn't have to be dynamic, as I think it's reasonable to assume the list of post types active on a given site isn't very long.

Reply:
Good idea, @lukecarbis. Would this be a dropdown of all registered post types, like from [get_post_types()](https://codex.wordpress.org/Function_Reference/get_post_types)?

Reply:
Exactly, yes.

Reply:
Sounds good, @lukecarbis. And it would be in the `/wp-admin` > Block Lab UI, not the block editor, right?

![dropdown-post-types](https://user-images.githubusercontent.com/4063887/56170814-94f8f980-5fa8-11e9-8db6-693d47a9e6b9.jpg)

Reply:
Yes – exactly. Then in the block editor the posts would be filtered by the post type you selected here.

Reply:
Good idea. This is implemented now.